spec/features/search/response_search_spec.rb

```diff
@@ -41,4 +42,37 @@
       "Error: Your search query could not be understood due to unexpected text near the end."
     )
   end
+
+  scenario "search filters" do
+    login(user)
+    visit "/en/m/#{mission.compact_name}/responses"
+
+    search_for(%(foo))
+    expect(page).to have_field("search", with: "foo")
+
+    new_search_for(%(form:"Form 1"))
+    expect(page).to have_field("search", with: "")
+    expect(page).to have_css("#form-filter.active-filter", text: "Form (Form 1)")
```

Comment:
Should we be covering somewhere in here that the search is actually being applied? i.e. that the correct rows are showing and not showing? Maybe not for all the searches (since we are covering correct search results at the model level) but for one of them?

Reply:
Done.